conda-recipes/llvmdev/build.sh

```diff
@@ -2,7 +2,7 @@ CMAKE_COMMON_VARIABLES=" -DCMAKE_INSTALL_PREFIX=$PREFIX \
     -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD=host \
     -DLLVM_INCLUDE_TESTS=OFF -DLLVM_INCLUDE_UTILS=OFF \
     -DLLVM_INCLUDE_DOCS=OFF -DLLVM_INCLUDE_EXAMPLES=OFF \
-    -DLLVM_ENABLE_TERMINFO=OFF \
+    -DLLVM_ENABLE_TERMINFO=OFF -DLLVM_ENABLE_ASSERTIONS=ON \
```

Comment:
This will be useful :-) I wonder at which point we dropped assertions? (probably by mistake)